org2jekyll.el

```diff
@@ -412,16 +384,13 @@ Return DEFAULT-VALUE if not found."
       data
     default-value))
 
-(defvar org2jekyll-header-metadata nil
-  "The needed headers for org buffer for org2jekyll to work.")
-
-(setq org2jekyll-header-metadata '(("title"       . 'mandatory)
-                                   ("date")
-                                   ("categories"  . 'mandatory)
-                                   ("tags")
-                                   ("description" . 'mandatory)
-                                   ("author")
-                                   ("layout"      . 'mandatory)))
+(setq org2jekyll-header-metadata '((:title       . 'mandatory)
```

Comment:
I liked the defvar, it's to avoid a warning at installation time.
I voluntarily declared it nil and set it after for dev reasons. When i was not yet sure of what i should put in there.

It should even be a defconst now.